sn_cli/src/subcommands/files/mod.rs

```diff
@@ -112,7 +112,8 @@ pub(crate) async fn files_cmds(
                 );
             }
 
-            let file_api: Files = Files::new(client, root_dir.to_path_buf());
+            let download_dir = dirs_next::download_dir().expect("Download directory is obtainable");
```

Comment:
Can we rework this to avoid a potential panic with the `expect` here? 

That appears to be the issue on this CI run: https://github.com/maidsafe/safe_network/actions/runs/7102691487/job/19333646110?pr=1039

(Although it's not clear to me why this is failing to find the dir 🤔 )